e2e/lockedFields.e2e.spec.ts

```diff
@@ -57,7 +57,9 @@ test.describe('Locked fields', () => {
 
   function saveUserProfile() {
     // If there's an error in this code, let it bubble up for the caller to deal with it.
-    deploymentProfile = readDeploymentProfiles();
```

Comment:
You can just do
```
deploymentProfile = await readDeploymentProfiles();
```
and then the comment at line 59 still holds true. This change throws away errors rather than having them bubble up.
And the caller to `saveUserProfile` is already async, so there's no problem making `saveUserProfile` async

Reply:
done